runtime/jcl/exports.cmake

```diff
@@ -519,7 +519,7 @@ if(JAVA_SPEC_VERSION EQUAL 8)
 endif()
 
 # java 9+
-if(NOT JAVA_SPEC_VERSION LESS 10)
+if(NOT JAVA_SPEC_VERSION LESS 9)
```

Comment:
Could just roll this into jdk11 since 9, 10 are obsolete and no longer supported.